src/ape/managers/converters.py

```diff
@@ -70,6 +70,18 @@ def convert(self, value: Any) -> int:
         return to_int(HexBytes(value))
 
 
+class HexListConverter(ConverterAPI):
+    """
+    Convert list of hex values to single concatenated hex value.
+    """
+
+    def is_convertible(self, value: Any) -> bool:
+        return isinstance(value, list) and all(is_hex(v) or isinstance(v, bytes) for v in value)
+
+    def convert(self, value: Any) -> bytes:
+        return HexBytes(b"".join([HexBytes(v) for v in value]))
```

Comment:
Technically don't need a list comprehension since `str.join` takes an iterable (saves you a list obj initialization)

```suggestion
        return HexBytes(b"".join(HexBytes(v) for v in value))
```